aws/k3s/main.tf

```diff
@@ -0,0 +1,67 @@
+provider "aws" {
+  profile = var.aws_profile
+  region = var.region
+}
+
+locals {
+  user_data = <<DATA
+#cloud-config
+
+runcmd:
+ - apt install curl systemd cgroup-bin cgroup-lite libcgroup1 -y
+ - curl -sfL https://get.k3s.io | sh -
+final_message: "Opsero k3s successfully created, after $UPTIME seconds"
+
+DATA
+}
+
```

Comment:
All of this code should be in the `aws` layer above. We want to have a single code and pick either k3s or eks. So